packages/ragbits-evaluate/src/ragbits/evaluate/dataset_generator/utils.py

```diff
@@ -0,0 +1,44 @@
+import ast
+import re
+import warnings
+from difflib import SequenceMatcher
+from itertools import combinations
+
+
+def get_closest_substring(long: str, short: str) -> str:
+    """
+    Finds the closest substring to short string in longer one
+    Args:
+        long: str - longer string
+        short: str - shorter string
+    Returns:
+        closest substring of longer
+    """
+    a, b = max(
+        combinations(re.finditer("|".join(short.split()), long), 2),
+        key=lambda c: SequenceMatcher(None, long[c[0].start() : c[1].end()], short).ratio(),
+    )
+    return long[a.start() : b.end()]
+
+
+def get_passages_list(raw_passages: str) -> list[str]:
+    """
+    Formats LLM output to list of passages
+    Args:
+        raw_passages: string representing raw passages returned by llm
+    Returns:
+        list of parsed passages
+    """
+    match = re.search(r"\[(.*?)\]", raw_passages, re.DOTALL)
+
+    if match:
+        passages_content = match.group(1)
+        try:
+            # Use eval to convert the string to a list, assuming it's a valid list-like format
+            return ast.literal_eval("[" + passages_content + "]")
```

Comment:
You shouldn't have to use `eval` for that, especially on content returned by an LLM.

Reply:
done